libr/core/panels.c

```diff
@@ -4970,6 +4970,38 @@ static void handle_tab_new(RCore *core) {
 	core->panels_root->n_panels++;
 }
 
+static void handle_tab_new_with_cur_panel (RCore *core) {
+	RPanels *panels = core->panels;
+	if (panels->n_panels <= 1) {
+		return;
+	}
+
+	RPanelsRoot *root = core->panels_root;
+	if (root->n_panels >= PANEL_NUM_LIMIT) {
+		return;
+	}
+
+	RPanel *cur = getCurPanel (panels);
+
+	root->n_panels++;
+	root->panels[root->n_panels - 1] = panels_new (core);
```

Comment:
Chk for nul

Reply:
fair enough. a few more points just occurred to me to fix in fact :)